ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -162,8 +172,8 @@ RUN git clone https://github.com/rupa/z /home/user/.z.sh \
 # Install the latest Vim.
 RUN mkdir /tmp/vim \
  && cd /tmp/vim \
- && curl -L https://github.com/vim/vim/archive/v8.0.1071.tar.gz | tar xz \
- && cd vim-8.0.1071/src \
+ && curl -L https://github.com/vim/vim/archive/v8.0.1171.tar.gz | tar xz \
+ && cd vim-8.0.1171/src \
```

Comment:
Don't know if it is feasible in Dockerfiles, but wouldn't you be better off having a set of environment variables defining which version number should be downloaded, and then use these version numbers in all the curl/cd/etc lines?

Reply:
Great suggestion! Thanks 😄 I'll look into using unique version variables.